implicits/src/com/rallyhealth/weepickle/v1/implicits/internal/Macros.scala

```diff
@@ -68,14 +68,17 @@ object Macros {
       omitDefault: Boolean,
       default: c.Tree,
       localTo: TermName,
-      aggregate: TermName
+      aggregate: TermName,
+      omitJson: Boolean
```

Comment:
This is definitely the right way to solve it, but you will hit the same binary compatibility issues that I hit the last time I attempted to change the Arguments structure. I have a refactoring in https://github.com/rallyhealth/weePickle/pull/24 that take all these structures private, but it includes other changes I don't think we want to make now. I'll remove those changes and just include the refactoring.

Reply:
Thanks, Russ. I'll hold off on begging for stamps for this PR until yours is in. 🙂 Then build off your changes.